homeassistant/components/tomorrowio/strings.json

```diff
@@ -61,6 +61,15 @@
           "freezing_rain": "Freezing Rain",
           "ice_pellets": "Ice Pellets"
         }
+      },
+      "uv_index": {
+        "state": {
+          "low": "Low",
+          "moderate": "Moderate",
+          "high": "High",
+          "very_high": "Very High",
```

Comment:
```suggestion
          "very_high": "Very high",
```

Reply:
I meant to contribute some translations anyway. I'm not 100% sure how it works. Is the stuff added in the strings.json automatically added to Lokalize and, when translated there, pushed to the repo?

Reply:
When a PR gets merged, all translation strings in Lokalize gets synced, so new ones get added (and removed ones, i dont know). And I think every release the latest translations get used